src/components/emails/WelcomeEmail.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Hr,
+  Html,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from "@react-email/components";
+import * as React from "react";
+import Logo from "@/components/Logo/index";
+
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : "";
+
+interface StripeWelcomeEmailProps {
+  firstName: string;
+  programName: string;
+}
+
+export const StripeWelcomeEmail: React.FC<StripeWelcomeEmailProps> = ({
```

Comment:
Remove references to Stripe

Also I would prefer to keep the components consistent in how they type stuff. so something like

```
export default function WelcomeEmail({name, programName}: WelcomeEmailProps) {...}